### PR TITLE
Add separate aperture for design matrix construction to PLDCorrector

### PR DESCRIPTION
**Pull request by nksaunders**
When applying the `PLDCorrector`, it is sometimes desired to use one aperture mask to generate the light curve and a separate aperture mask to select pixels to use as basis vectors in the PLD design matrix. `PLDCorrector.correct()` now takes two aperture mask arguments:

 - `aperture_mask`: pixels used in aperture photometry to generate the raw flux light curve
- `pld_aperture`: pixels used when selecting PLD basis vectors

Both of these can take the default lightkurve aperture_mask options: `array-like, 'pipeline', 'all', 'threshold', or None`. 

Thank you @christinahedges for this feature request! Do you think the `aperture_mask` argument should be renamed? I'm hesitant to veer away from the naming convention we use in other functions. Feedback welcome!

### CODE REVIEW

**Comment by barentsen** [comment]
To be more consistent with `aperture_mask`, I wonder if we should call the second parameter `pld_aperture_mask`.

File: lightkurve/correctors.py [hunk 1]
```diff
@@ -672,15 +680,18 @@ def correct(self, aperture_mask=None, cadence_mask=None, gp_timescale=30,
         self.flux_err = self.flux_err[nanmask]
         self.time = self.time[nanmask]
 
+        # parse PLD aperture mask
+        pld_pixels = self.tpf._parse_aperture_mask(pld_aperture)
```
Comment (barentsen) [inline lightkurve/correctors.py]: :point_up: It may be best to consistently use the suffix `_mask` in all variables that contain a ... mask.

Comment (barentsen) [reply lightkurve/correctors.py]: (A reader of the code may be tempted to think that `pld_pixels` contain pixel values otherwise.)

**Comment by barentsen** [comment]
I left some thoughts above.  Two more items of feedback are:
* A PR re-factoring `correctors.py` into a sub-package has just been merged (#519).  It will be hard to resolve the merging conflict in this PR, so it may be easiest to pull keplergo/master and start a new branch from master.
* Let's add a unit test, even if all it does is pass some mask to the new parameter.

**Comment by nksaunders** [comment]
@barentsen got it 👍 closing and reopening this PR to avoid merge conflicts.

**Pull request closed without merging**